eze-nonso/ES6-Classes-ezeoke-chinonso
# Ft my oop class #150647328

#### What does this PR do?
Shows es6 OOP real world implementation 
#### Description of Task to be completed?
#### How should this be manually tested?
Create objects of class Bulldozer and Sports and test for parent method inheritance. Also attempt to verify parent class Vehicle is abstract
#### Any background context you want to provide?
I used a Chrome console to test-run
#### What are the relevant pivotal tracker stories?
 #150647328
#### Screenshots (if appropriate)
#### Questions:
